oluseyianani/stackoverflow-lite-react
# #162336003 Signup a User to Stackoverflow-Lite

#### What does this PR do?
Registers a user to the platform

#### Description of Task to be completed?
- Signup components
- Testing for signup feature

#### How should this be manually tested?
After cloning this repo, cd into it and install all depencencies.
Then run npm run dev to start the application of development mode.
Click on signup and fill in the fields to see its responses.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#162336003](https://www.pivotaltracker.com/story/show/162336003)
#### Screenshots (if appropriate)
<img width="924" alt="screenshot 2018-12-31 at 6 18 59 am" src="https://user-images.githubusercontent.com/25327188/50554933-5030a780-0cc4-11e9-85ac-37d3042dcf59.png">

<img width="926" alt="screenshot 2018-12-31 at 6 20 16 am" src="https://user-images.githubusercontent.com/25327188/50554935-5cb50000-0cc4-11e9-8675-30075f6c9741.png">

#### Questions:
N/A
